### ISSUE TITLE
Must know about Timothy the Monkey

### ISSUE THREAD

**Issue by thebinarypenguin** [balderdashy/sails-docs#400]
In the file `concepts/ORM/Models.md` (currently line 91) you link to an incident involving Timothy the Monkey, but the link is empty.

What is so special about this monkey? I need closure!

**Comment by jub3i** [comment]
@mikermcneil Whats the story with this monkey business? :) Seriously though, should I remove it from the docs?

**Comment by rachaelshaw** [comment]
Looking into it. :banana: 
cc @uncletammy :monkey:

**Comment by tjwebb** [comment]
the suspense is killing me!

**Comment by brettimus** [comment]
+1000

**Comment by Duckbrain** [comment]
I found him! [Timothy the Monkey's Story `/concepts/ORM/Models.md#L278`](https://github.com/balderdashy/sails-docs/commit/eb530f1c963766ab49be05b33b789929669f3c76#diff-a5079fc6f7264e7f7021b77b4a455240R238) It is hidden in a comment on the page. If you view source you can see it. Do we want to take out the reference (and probably the comment) or work it back into the page?

**Comment by brettimus** [comment]
I see it! I see it! Thanks for pointing out. I created a PR to remove references. Waiting for someone else to sign off on it. I'll be sad to see him go.

**Comment by sailsbot** [comment]
Thanks for posting, @thebinarypenguin.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by brettimus** [comment]
Hello, Timothy. We meet again.

**Comment by sailsbot** [comment]
Thanks for posting, @thebinarypenguin.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sailsbot** [comment]
Thanks for posting, @thebinarypenguin.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sailsbot** [comment]
Thanks for posting, @thebinarypenguin.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by thebinarypenguin** [comment]
Why does this keep getting reopened?

**Comment by mikermcneil** [comment]
@thebinarypenguin this is the single most important issue in the history of the framework.

**Comment by brettimus** [comment]
Yo team I got this one.

**Comment by brettimus** [comment]
True story, I had a dream about a monkey named Timothy the other week. 

_**w h o a**_

:open_mouth:

**Comment by thebinarypenguin** [comment]
I feel like I've created a monster.

**Comment by mikermcneil** [comment]
> True story, I had a dream about a monkey named Timothy the other week.

... wait.  What day was it....??!!!!!

**Comment by mikermcneil** [comment]
@thebinarypenguin au contraire, my feathered friend.  Timothy is no monster.

**Comment by brettimus** [comment]
@mikermcneil I can't remember the exact day, but I do remember that he kinda looked like this:

![timothy](https://cloud.githubusercontent.com/assets/3201327/13031325/191e47d4-d27f-11e5-861c-ac93446ce939.JPG)

**Comment by FerRubioMorales** [comment]
Lol guys, you all are crazy😂

**Comment by sailsbot** [comment]
@thebinarypenguin,@jub3i,@rachaelshaw,@tjwebb,@brettimus,@duckbrain,@sailsbot,@mikermcneil,@FerRubioMorales: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by brettimus** [comment]
Give it up, @sailsbot

**Comment by derek-webseeds** [comment]
I'm here because the monkey lives.

**Comment by sailsbot** [comment]
@thebinarypenguin,@jub3i,@rachaelshaw,@tjwebb,@brettimus,@duckbrain,@sailsbot,@mikermcneil,@FerRubioMorales,@derek-webseeds: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by animedbz16** [comment]
It is amazing to me that this issue receives more attention and is reopened
so many times while actual issues receive little to no feedback and
ultimately end with sailsbot auto closing them.

**Comment by sailsbot** [comment]
@thebinarypenguin,@jub3i,@rachaelshaw,@tjwebb,@brettimus,@duckbrain,@sailsbot,@mikermcneil,@FerRubioMorales,@derek-webseeds,@animedbz16: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by TimOgilvy** [comment]
oh sailsbot.  Happy news, there is still a monkey. I found him in the 1.0 docs.

Shhh don't tell anyone, maybe we can sneak in some 🍌 🍌 's  and keep him alive.

Don't let the monkey die, or PETA will be here protesting.
